Makefile

```diff
@@ -34,7 +34,9 @@ docker-push:
 	docker push twilio/twilio-ruby:${CURRENT_TAG}
 
 cluster-test:
-	bundle exec rspec ./cluster_spec.rb
+	bundle exec rspec ./cluster/cluster_spec.rb
```

Comment:
instead of keeping three different statements here, we can just keep `bundle exec rspec ./cluster` and it will work